symphony/lib/toolkit/class.xsrf.php

```diff
@@ -23,7 +23,11 @@ class XSRF
      */
     public static function getSessionToken()
     {
-        $token = $_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'];
+        $token = null;
+
+        if (isset($_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'])) {
+            $token = $_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'];
+        }
```

Comment:
Is there really no other way to avoid the duplication (i.e. `$_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token']`) here ? We must have that pattern everywhere...